students/cfarris/lessons/lesson03/assignment/basic_operations.py

```diff
@@ -78,14 +79,35 @@ def update_customer_credit(customer_id, credit_limit):
     A ValueError will be raised if customer does not exist in database.
     '''
     try:
-       retr_customer = Customer.get(Customer.customer_id == customer_id)
+        retr_customer = Customer.get(Customer.customer_id == customer_id)
+        retr_customer.credit_limit = credit_limit
+        retr_customer.save()
+        if credit_limit < 0:
+            logging.warning('credit_limit parameter is ',
+                            'less than 0: ', credit_limit)
+    except ValueError as err:
+        logging.error('error replacing customers credit limit: ,', err)
+        error_statement = 'An error occured while replacing customers credit.\
+                         Credit unchanged.'
```

Comment:
You can't be sure it wasn't changed. What happens when the error occurred when comparing credit_limit to 0? In that case, the error happened after the credit was saved.